src/components/profile-view/profile-view.jsx

```diff
@@ -0,0 +1,183 @@
+import { useState } from "react";
+import { MovieCard } from "../MovieCard/movie-card";
+import Button from "react-bootstrap/Button";
+import Col from "react-bootstrap/Col";
+import Form from "react-bootstrap/Form";
+import Row from "react-bootstrap/Row";
+import Modal from "react-bootstrap/Modal";
+import "./profile-view.scss";
+
+export const ProfileView = ({ token, setUser, movies }) => {
+  const user = JSON.parse(localStorage.getItem("user"));
+  const [username, setUsername] = useState(user.Username);
+  const [password, setPassword] = useState("");
+  const [email, setEmail] = useState(user.Email);
+  const [birthday, setBirthday] = useState(user.Birthday);
+  const [showModal, setShowModal] = useState(false);
+  const favoriteMovies = movies?.filter((movie) =>
+    user.FavoriteMovies.includes(movie._id)
+  );
+
+  const handleShowModal = () => setShowModal(true);
+  const handleCloseModal = () => setShowModal(false);
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+
+    let data = {
+      Username: username,
+      Email: email,
+      Birthday: birthday,
+    };
+    if (password) {
+      data["Password"] = password;
+    }
+
+    fetch(`https://my-flix-host.onrender.com/users/${user.Username}`, {
```

Comment:
Updating the user profile infos doesn't work, it returns 'user doesn't exist' error. Which means that there is an issue in the url. Check the API documentation, I think you're supposed to pass the user id not the name.